Crash report 2.8 - InstantiationException in ThemeTabFragment

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.themes.ThemeBrowserActivity}: android.support.v4.app.Fragment$InstantiationException: Unable to instantiate fragment org.wordpress.android.ui.themes.ThemeTabFragment: make sure class name exists, is public, and has an empty constructor that is public
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1735)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:1754)
       at android.app.ActivityThread.access$1500(ActivityThread.java:156)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1000)
       at android.os.Handler.dispatchMessage(Handler.java:130)
       at android.os.Looper.loop(SourceFile:351)
       at android.app.ActivityThread.main(ActivityThread.java:3821)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:538)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:969)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:727)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: android.support.v4.app.Fragment$InstantiationException: Unable to instantiate fragment org.wordpress.android.ui.themes.ThemeTabFragment: make sure class name exists, is public, and has an empty constructor that is public
       at android.support.v4.app.Fragment.instantiate(Fragment.java:413)
       at android.support.v4.app.FragmentState.instantiate(Fragment.java:97)
       at android.support.v4.app.FragmentManagerImpl.restoreAllState(FragmentManager.java:1790)
       at android.support.v4.app.FragmentActivity.onCreate(FragmentActivity.java:213)
       at org.wordpress.android.ui.WPActionBarActivity.onCreate(WPActionBarActivity.java:121)
       at org.wordpress.android.ui.themes.ThemeBrowserActivity.onCreate(ThemeBrowserActivity.java:67)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1082)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1699)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:1754)
       at android.app.ActivityThread.access$1500(ActivityThread.java:156)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1000)
       at android.os.Handler.dispatchMessage(Handler.java:130)
       at android.os.Looper.loop(SourceFile:351)
       at android.app.ActivityThread.main(ActivityThread.java:3821)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:538)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:969)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:727)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: java.lang.InstantiationException: org.wordpress.android.ui.themes.ThemeTabFragment
       at java.lang.Class.newInstanceImpl(Class.java)
       at java.lang.Class.newInstance(Class.java:1440)
       at android.support.v4.app.Fragment.instantiate(Fragment.java:402)
       at android.support.v4.app.FragmentState.instantiate(Fragment.java:97)
       at android.support.v4.app.FragmentManagerImpl.restoreAllState(FragmentManager.java:1790)
       at android.support.v4.app.FragmentActivity.onCreate(FragmentActivity.java:213)
       at org.wordpress.android.ui.WPActionBarActivity.onCreate(WPActionBarActivity.java:121)
       at org.wordpress.android.ui.themes.ThemeBrowserActivity.onCreate(ThemeBrowserActivity.java:67)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1082)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1699)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:1754)
       at android.app.ActivityThread.access$1500(ActivityThread.java:156)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1000)
       at android.os.Handler.dispatchMessage(Handler.java:130)
       at android.os.Looper.loop(SourceFile:351)
       at android.app.ActivityThread.main(ActivityThread.java:3821)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:538)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:969)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:727)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

--- a/src/org/wordpress/android/ui/themes/ThemeSearchFragment.java
+++ b/src/org/wordpress/android/ui/themes/ThemeSearchFragment.java
@@ -24,12 +24,8 @@ public class ThemeSearchFragment extends ThemeTabFragment implements OnQueryText
     public static final String TAG = ThemeSearchFragment.class.getName();
     private static final String BUNDLE_LAST_SEARCH = "BUNDLE_LAST_SEARCH";
 
-    protected ThemeSearchFragment(int page) {
-        super(page);
-    }
-
     public static ThemeSearchFragment newInstance() {
-        ThemeSearchFragment fragment = new ThemeSearchFragment(-1);
+        ThemeSearchFragment fragment = new ThemeSearchFragment();
 
         Bundle args = new Bundle();
         args.putInt(ARGS_SORT, ThemeSortType.POPULAR.ordinal());

--- a/src/org/wordpress/android/ui/themes/ThemeTabFragment.java
+++ b/src/org/wordpress/android/ui/themes/ThemeTabFragment.java
@@ -59,6 +59,7 @@ public class ThemeTabFragment extends SherlockFragment implements OnItemClickLis
     }
 
     protected static final String ARGS_SORT = "ARGS_SORT";
+    protected static final String ARGS_PAGE = "ARGS_PAGE";
     protected static final String BUNDLE_SCROLL_POSTION = "BUNDLE_SCROLL_POSTION";
 
     protected GridView mGridView;
@@ -66,18 +67,14 @@ public class ThemeTabFragment extends SherlockFragment implements OnItemClickLis
     protected ThemeTabAdapter mAdapter;
     protected ThemeTabFragmentCallback mCallback;
     protected int mSavedScrollPosition = 0;
-    protected int mPage;
     private boolean mShouldRefreshOnStart;
     private PullToRefreshHelper mPullToRefreshHelper;
 
-    protected ThemeTabFragment(int page) {
-        mPage = page;
-    }
-
     public static ThemeTabFragment newInstance(ThemeSortType sort, int page) {
-        ThemeTabFragment fragment = new ThemeTabFragment(page);
+        ThemeTabFragment fragment = new ThemeTabFragment();
         Bundle args = new Bundle();
         args.putInt(ARGS_SORT, sort.ordinal());
+        args.putInt(ARGS_PAGE, page);
         fragment.setArguments(args);
         return fragment;
     }
@@ -115,7 +112,7 @@ public class ThemeTabFragment extends SherlockFragment implements OnItemClickLis
                         return;
                     }
                     if (getActivity() instanceof ThemeBrowserActivity) {
-                        ((ThemeBrowserActivity) getActivity()).fetchThemes(mPage);
+                        ((ThemeBrowserActivity) getActivity()).fetchThemes(getArguments().getInt(ARGS_PAGE));
                     }
                 }
             });

